### PR TITLE
issue 1328 circle scope updates again when events attr is not subscribed

### PR DESCRIPTION
**Pull request by nmccready**
fix(circle): scope properties update again even if the event attribute is not set in the directive

issue /#1328 fixed

### CODE REVIEW

File: spec/coffee/directives/api/circle.spec.coffee [hunk 1]
```diff
@@ -108,6 +108,44 @@ describe 'uiGmapCircle', ->
...
           , 500
 
+      it 'change center if directive events is not defined', (done) ->
+        #issue 1271
+        @html = """
+          <ui-gmap-google-map draggable="true" center="map.center" zoom="map.zoom">
+              <ui-gmap-circle center='map.circle.center'
+                            radius='map.circle.radius'
+                            fill='map.circle.fill'
+                            stroke='map.circle.stroke'
```
Comment (nmccready) [inline spec/coffee/directives/api/circle.spec.coffee]: no event attribute set which without this fix was failing

File: src/coffee/directives/api/models/parent/circle-parent-model.coffee [hunk 2]
```diff
@@ -42,43 +42,42 @@ angular.module('uiGmapgoogle-maps.directives.api.models.parent')
...
       clean()
-      @listeners = @setEvents gObject, scope, scope, ['radius_changed']
-      if @listeners?
-        @listeners.push google.maps.event.addListener gObject, 'radius_changed', ->
-          ###
-            possible google bug, and or because a circle has two radii
-            radius_changed appears to fire twice (original and new) which is not too helpful
-            therefore we will check for radius changes manually and bail out if nothing has changed
-          ###
+      @listeners = @setEvents(gObject, scope, scope, ['radius_changed']) or [] #or needed incase no-one is listening in controller
```
Comment (nmccready) [inline src/coffee/directives/api/models/parent/circle-parent-model.coffee]: main problem here `@listeners` needs to be set to [] even if scope.events is null

**Comment by nmccready** [comment]
@cthrax @wrummler

**Comment by nmccready** [comment]
merging

**Comment by cthrax** [comment]
:+1:

**Pull request merged**